cypress/commons/utils/fileUtils.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const parseMultipartFormData = (dataString) => {
+  const boundary = dataString.split('\r\n')[0];
+  let formParts = dataString.split(boundary);
+  formParts.pop();
+  formParts.shift();
+  formParts = formParts.map((el) => (el = el.replace(/\r\n$/, '')));
+  formParts = formParts.map((el) => (el = el.replace(/^\r\n/, '')));
+
+  const form = {};
+  formParts.forEach((formPart) => {
+    const regexMatch = formPart.match(/Content-Disposition: form-data; name="(\w*)"(?:[\S\s]*)?\r\n\r\n([\S\s]*)/);
+    const parameterName = regexMatch[1];
+    const parameterValue = regexMatch[2];
```

Comment:
Just a suggestion: maybe we can use `regexMatch[2]` directly, without assigning it to a variable? Also I'd say that `regexMatch` is too generic, probably something like `parameterByRegex` or `parameterMatch` could be more explicit ? :thinking:

Reply:
I'm rather for keeping `parameterName` because of better readability but it's your call :grin: